packages/generator/src/edmx-to-vdm/common/action-function-return-types.ts

```diff
@@ -135,7 +135,7 @@ function getEdmReturnType(
   extractResponse: ExtractResponse,
   oDataVersion: ODataVersion
 ): VdmFunctionImportReturnType {
-  const typeMapping = getTypeMappingActionFunction(edmType);
+  const typeMapping = getTypeMappingActionFunction(edmType, true);
```

Comment:
see above

Reply:
Here, I think we don't have the information whether this is nullable or not...